ci/prebuild.yml

```diff
@@ -15,30 +21,35 @@ style prebuild:
   script:
     - nix-shell --run "poetry run make style_check"
 
+# Check validity of coin definitions and protobuf files
 common prebuild:
   stage: prebuild
   <<: *gitlab_caching
   script:
     - nix-shell --run "poetry run make defs_check"
 
+# Check validity of auto-generated files
 gen prebuild:
   stage: prebuild
   <<: *gitlab_caching
   script:
     - nix-shell --run "poetry run make gen_check"
 
+# Checking format of .editorconfig files
 editor prebuild:
   stage: prebuild
   <<: *gitlab_caching
   script:
     - nix-shell --run "make editor_check"
 
+# All .yml/.yaml files are checked for syntax validity and other correctness
 yaml prebuild:
   stage: prebuild
   <<: *gitlab_caching
   script:
     - nix-shell --run "poetry run make yaml_check"
 
+# Checking the format of release commit messages
```

Comment:
Nit: I understand you didn't write it but can you please ensure the sentences end with a `.`?